packages/velo-external-db-core/lib/service/schema_matchers.js

```diff
@@ -9,10 +9,20 @@ const appendAllowedOperationsToDbs = (dbs, allowedSchemaOperations) => {
     }))
 }
 
-const toHaveSchemas = ( dbsWithAllowedOperations ) => ({ schemas: dbsWithAllowedOperations.map(asWixSchema) })
+const toHaveSchemas = ( collections, func, ...args ) => ({
```

Comment:
`func` is a match function for one collection, it needs to return a matcher (for example we want to test that each collection has a property called `allowedOperations`)